src/Documentation/Markdown/Markdown.js

```diff
@@ -209,14 +218,18 @@ export default class Markdown extends React.PureComponent {
           astPlugins={[linker()]}
         />
         <NavigationButtons>
-          <Button onClick={() => onNavigate(prev)} disabled={!prev}>
-            <i className="prev" />
-            <span>Prev</span>
-          </Button>
-          <Button onClick={() => onNavigate(next)} disabled={!next}>
-            <span>Next</span>
-            <i className="next" />
-          </Button>
+          <NextLink href={PAGE_DOC} as={prev} passHref>
+            <Button disabled={!prev}>
+              <i className="prev" />
+              <span>Prev</span>
+            </Button>
+          </NextLink>
```

Comment:
...or maybe the collapsed navigation without js IS related to this PR?

Reply:
I think the previous version was not working w/o JS at all, btw